src/types.ts

```diff
@@ -2122,6 +2122,19 @@ export type User<UserType = UnknownType> = UserType & {
   username?: string;
 };
 
+export type DeleteType = 'soft' | 'hard';
+
+export type DeleteUserOptions = {
+  user_ids: string[];
+  conversations?: DeleteType;
+  messages?: DeleteType;
+  user?: DeleteType;
```

Comment:
do we skip deleting these if no option is passed?

Reply:
actually, I think `user` should not be optional, it will be set to default on the backend anyway

Reply:
```suggestion
  user?: DeleteType;
  conversations?: DeleteType;
  messages?: DeleteType;
```